mcp/communications/evtm.c

```diff
@@ -225,7 +226,7 @@ void evtm_compress_and_send(struct evtmInfo *evtm_info) {
     if (setup_EVTM_config(evtm_info, &evtm_setup) != 0) {
         blast_fatal("EVTM setup failed");
     }
-    while (!testing_evtm()) {
-        infinite_loop_EVTM(&evtm_setup);
+    while (!enable_EVTM_loop()) {
```

Comment:
Oof, sorry, this is more confusing now. I think the function name should stay similar to this, but it should return a 1 when not mocked, and the while loop should be conditioned on `enable_loop()` instead of `!enable_loop()`. This will require updating UT code, I think.

Reply:
whoops you are right